ingestion/src/metadata/ingestion/source/singlestore.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from metadata.ingestion.ometa.openmetadata_rest import MetadataServerConfig
+from metadata.ingestion.source.sql_source import SQLSource
+from metadata.ingestion.source.sql_source_common import SQLConnectionConfig
+
+
+class SingleStoreConfig(SQLConnectionConfig):
+    host_port = "localhost:3306"
+    scheme = "mysql+pymysql"
+    service_type = "SingleStore"
```

Comment:
Maybe this would need a change on the source SQL config info, but could we instead of hardcode the service type here use the generated class? `DatabaseServiceType.SingleStore`, as we have this defined properly in the JSON Schemas.

Thanks!